gio/Gir.toml

```diff
@@ -284,6 +284,24 @@ generate_builder = true
     name = "run"
     manual = true
     doc_trait_name = "ApplicationExtManual"
+    [[object.function]]
+    name = "hold"
+    # Returns an value that releases on Drop
+    manual = true
+    doc_trait_name = "ApplicationExtManual"
+    [[object.function]]
+    name = "release"
+    manual = true
+    doc_trait_name = "ApplicationExtManual"
+    [[object.function]]
+    name = "mark_busy"
+    manual = true
+    # Returns an value that unmarks-busy on Drop
+    doc_trait_name = "ApplicationExtManual"
+    [[object.function]]
+    name = "unmark_busy"
+    manual = true
```

Comment:
What's the reason for this one?

Reply:
See 4 lines above

Reply:
And how someone without this PR context can guess it? ;)